apps/src/templates/studioHomepages/CoteacherInviteNotification.jsx

```diff
@@ -18,15 +18,34 @@ export const showCoteacherInviteNotification = coteacherInvite => {
   return !!coteacherInvite && DCDO.get('show-coteacher-ui', true);
 };
 
+export const showCoteacherForPlInviteNotification = coteacherInviteForPl => {
+  return !!coteacherInviteForPl && DCDO.get('show-coteacher-ui', true);
+};
+
 const CoteacherInviteNotification = ({
+  isForPl,
   asyncLoadCoteacherInvite,
   asyncLoadSectionData,
   coteacherInvite,
+  coteacherInviteForPl,
 }) => {
-  if (!showCoteacherInviteNotification(coteacherInvite)) {
+  if (
+    !(
```

Comment:
nit: This is entirely correct, I personally have a preference for `!a && !b` instead of `!(a || b)`. I think it's a bit easier to read, but totally optional and up to you.